File: app/src/main/java/com/mytlogos/enterprise/Formatter.java
package com.mytlogos.enterprise;

import org.joda.time.DateTime;
import org.joda.time.Instant;
import org.joda.time.format.DateTimeFormat;

public class Formatter {
    public static String formatDateTime(DateTime date) {
        return DateTimeFormat.forPattern("dd.MM.YYYY HH:mm").print(date);
    }

    public static DateTime parseDateTime(String isoDate) {
        return Instant.parse(isoDate).toDateTime();
    }
}

File: app/src/main/java/com/mytlogos/enterprise/Formatter.kt
package com.mytlogos.enterprise

import org.joda.time.DateTime
import org.joda.time.Instant
import org.joda.time.format.DateTimeFormat

fun DateTime.formatDateTime(): String {
    return DateTimeFormat.forPattern("dd.MM.YYYY HH:mm").print(this)
}

fun DateTime.formatDateTimeIso(): String {
    return this.toInstant().toString()
}

fun String.parseDateTime(): DateTime {
    return Instant.parse(this).toDateTime()
}